bundle/Resources/views/content_view_fallback/to_ez_platform/embed.html.twig

```diff
@@ -0,0 +1,16 @@
+{# content \Netgen\EzPlatformSiteApi\API\Values\Content #}
+{# location \Netgen\EzPlatformSiteApi\API\Values\Location #}
+
+{% if ezpublish.configResolver.getParameter('ng_fallback_without_subrequest') == true %}
+    {{ ng_ez_view_content_embedded(view_type, { "content": content.innerContent, "location": location.innerLocation }) }}
+{% else %}
+    with subrequest {{ render(
```

Comment:
Is `with subrequest` here a typo?

Reply:
A leftover from testing :) Removed in https://github.com/netgen/ezplatform-site-api/pull/155/commits/e6465d97407f9b5af02e3038859c98dce775f309